public/scripts/histogram.js

```diff
@@ -58,7 +63,20 @@ function getHistogram(fromYear, fromMonth, fromDate, toYear, toMonth, toDate, da
 	      .attr("transform", function(d) {
 			  return "translate(" + x(d.x0) + "," + y(d.length) + ")"; })
 	      .attr("width", function(d) { return x(d.x1) - x(d.x0) -1 ; })
-	      .attr("height", function(d) { return height - y(d.length); });
+	      .attr("height", function(d) { return height - y(d.length); })
+ 	      .on("mouseover", function(d) {    
+		    div .transition()    
+			.duration(200)    
+			.style("opacity", .8);    
+		    div .html("Number of Mementos: " + d.length + "<br/>" + d.x0.toString().substring(4,7) + " " + d.x0.toString().substring(11,15))  

```

Comment:
@a-mabe I am not too confident about `d.x0.toString().substring(4,7)` and `d.x0.toString().substring(11,15)`. What is in `d.x0` here? And how reliable is it when the number of mementos vary?

Reply:
d.length gets the number of mementos for that specific bar, I've tested this with sites that have over 800 mementos and some with less than 20. d.x0 contains a variable of type object which holds the date for that bar. To extract the month and year from the object to be displayed, I've used .toString().substring(4,7) and .toString().substring(11,15).